builtins/en/facets/instructions/fix.md

```
@@ -43,5 +45,9 @@ If you disputed any findings, include `## Disputed Findings` (follow the format
 | new (fixed in this iteration) | {N} |
 | reopened (recurrence fixed) | {N} |
 | persists (carried over, fixed in this iteration) | {N} |
+## Family coverage
+| family_tag | enumerated branches/inputs | covering test | status / reason |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)